### DISCUSSION TITLE
How can I reply to a message?

### DISCUSSION THREAD

**Discussion by RahulVerma989** [php-telegram-bot/core#1429] [Q&A]
Hi there!

I wanted my bot to reply to user's message. How can I do that?

Currently I am sending message like this.

BotRequest::sendMessage(['chat_id' => $chatId, 'text' => $text]);

This will send the message to the chat but I don't know how to reply to a specific message in the chat. It's not mentioned in the document as well.

**Comment by TiiFuchs** [comment]
It is mentioned here: https://core.telegram.org/bots/api#sendmessage

You need to use the reply_to_message_id parameter to specify the corresponding message_id from your sent or received message.
